.github/workflows/get-previous-release/action.yml

```diff
@@ -5,16 +5,23 @@ inputs:
   CHANNEL:
     required: true
     description: "The channel to get the version to (ga, cloud, beta)."
+  POSTFIX:
+    required: false
+    description: "The postfix that will be added to the package version -rc / -test / empty string."
 
 runs:
   using: "composite"
   steps:
     - shell: bash
       run: |
         # If the channel is either "cloud" or "beta", fetch all Git tags matching the channel pattern,
-        # excluding any "-rc" tags, and store them in the 'tags' variable. e.g. "refs/tags/v3.11.0-cloud1 , refs/tags/v3.11.0-cloud2"
+        # excluding any "-rc" and "-test" tags, and store them in the 'tags' variable. e.g. "refs/tags/v3.11.0-cloud1 , refs/tags/v3.11.0-cloud2"
         if [[ ${{ inputs.CHANNEL }} == "cloud" || ${{ inputs.CHANNEL }} == "beta" ]]; then
-           tags=$(git ls-remote --tags | grep ${{ inputs.CHANNEL }} | grep -v "\-rc" | awk '{print $2}')
+          if [[ "${{ inputs.POSTFIX }}" != "-test" ]]; then
+            tags=$(git ls-remote --tags | grep ${{ inputs.CHANNEL }} | grep -v "\-rc" | grep -v "\-test" | awk '{print $2}')
+          elif [[ "${{ inputs.POSTFIX }}" == "-test" ]]; then
```

Comment:
```suggestion
          else
```

Reply:
Done, 
I've changed the condition so the `if` will check the positive scenario.